languages/python/oso/tests/test_polar_roles.py

```diff
@@ -1021,10 +998,9 @@ def test_grandparent_child_role_perm(init_oso, sample_data):
         role_allow(actor, action, resource);
     """
     oso.load_str(policy)
+    oso.validate_config()
     # TODO: big red button to reset roles policy?
     # oso.roles.config = None
```

Comment:
now that we aren't persisting config on the Oso instance we can probably rm this pair of lines (and the other places it appears in this test suite)
```suggestion
```

ETA: will push a commit for this